ui/client/components/graph/node-modal/editors/expression/Duration/TimeRangeSection.tsx

```diff
@@ -29,8 +29,8 @@ export default function TimeRangeSection(props: Props) {
         {
           components.map(component => (
             <TimeRangeComponent
-              key={component.fieldName}
-              component={component}
+              key={component}
```

Comment:
key and timeRangeComponent with the same value?

Reply:
yes, now component is just enumerated name, I changes const and property names to be more clear